extensionsGallery.json

```diff
@@ -1622,14 +1622,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.3.9",
-							"lastUpdated": "11/5/2019",
+							"version": "0.4.1",
+							"lastUpdated": "03/08/2020",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/managed-instance-dashboard/managed-instance-dashboard-0.3.9.vsix"
+									"source": "https://github.com/JocaPC/AzureDataStudio-Managed-Instance/releases/download/0.4.1/managed-instance-dashboard-0.4.1.vsix"
```

Comment:
For direct VSIX installs we use the blob storage account. I'll handle uploading the file - just change the URL to `https://sqlopsextensions.blob.core.windows.net/extensions/managed-instance-dashboard/managed-instance-dashboard-0.4.1.vsix`

Reply:
I changed the path. File is in releases: https://github.com/JocaPC/AzureDataStudio-Managed-Instance/releases/download/0.4.1/managed-instance-dashboard-0.4.1.vsix